### PR TITLE
133 connection status very buggy

### LINKED ISSUE #133: connection status very bugy
### Content
- When the Front-end gets disconnected from the broker, the connection status remains true even though it should be false.
- When another front-end disconnects, the status on the front-end that is connected should remain true
- After resetting all connection are set to false
- When front-end/back-end disconnects, all devices should display no connection
- Add colour to the `✔` and `✖` 

### Feature
Front-end should display a non-connection status when not connected to the broker

### Action plan
- [x] find the cause of the bugs
- [x] fix them

### PR DESCRIPTION
**Pull request by mrlomar**
## Contents
Connection status now correct when:
- the broker disconnects
- the back-end disconnects
- another front-end disconnects
- after resetting

Add colour to the ✔ and ✖
## Issue
Closes #133

### CODE REVIEW

File: front-end/src/app/components/device/device.component.html [hunk 1]
```diff
@@ -1,32 +1,33 @@
...
+        <th mat-header-cell *matHeaderCellDef mat-sort-header class="custom-md-table-header"> Apparaat</th>
         <td mat-cell *matCellDef="let element"> {{ element.id }} </td>
       </ng-container>
       <ng-container matColumnDef="connection">
-        <th mat-header-cell *matHeaderCellDef mat-sort-header class="custom-md-table-header"> Connectie </th>
+        <th mat-header-cell *matHeaderCellDef mat-sort-header class="custom-md-table-header"> Connectie</th>
         <td mat-cell *matCellDef="let element" class="boolean-column">
-          <ng-container *ngIf="element.connection">&#10004;</ng-container>
-          <ng-container *ngIf="!element.connection">&#10006;</ng-container>
+          <ng-container *ngIf="element.connection"><div class="check-mark">&#10004;</div></ng-container>
```
Comment (IssaHanou) [inline front-end/src/app/components/device/device.component.html]: these should be `<span>`s

File: front-end/src/app/components/device/device.component.html [hunk 2]
```diff
@@ -1,32 +1,33 @@
 <div class="home-box device-box">
   <h3 class="header">Apparaten</h3>
   <div class="contents">
-    <table mat-table matSort #DeviceTableSort="matSort" [dataSource]="getDeviceList()" class="mat-elevation-z8 custom-md-table">
+    <table mat-table matSort #DeviceTableSort="matSort" [dataSource]="getDeviceList()"
+           class="mat-elevation-z8 custom-md-table">
       <ng-container matColumnDef="id">
-        <th mat-header-cell *matHeaderCellDef mat-sort-header class="custom-md-table-header"> Apparaat </th>
+        <th mat-header-cell *matHeaderCellDef mat-sort-header class="custom-md-table-header"> Apparaat</th>
```
Comment (IssaHanou) [inline front-end/src/app/components/device/device.component.html]: what happened with the spaces? either before and after or none - none is probably better

File: front-end/src/app/components/puzzle/puzzle.component.html [hunk 3]
```diff
@@ -9,8 +9,8 @@ <h3 class="header">Puzzels</h3>
       <ng-container matColumnDef="status">
         <th mat-header-cell *matHeaderCellDef mat-sort-header class="custom-md-table-header"> Status </th>
         <td mat-cell *matCellDef="let element" class="boolean-column">
-          <ng-container *ngIf="element.status">&#10004;</ng-container>
-          <ng-container *ngIf="!element.status">&#10006;</ng-container>
+          <ng-container *ngIf="element.status"><div class="check-mark">&#10004;</div></ng-container>
```
Comment (IssaHanou) [inline front-end/src/app/components/puzzle/puzzle.component.html]: again use `<span>`

File: front-end/src/app/components/device/device.component.html (hunk 1 above)
Comment (mrlomar) [reply front-end/src/app/components/device/device.component.html]: done

File: front-end/src/app/components/puzzle/puzzle.component.html (hunk 3 above)
Comment (mrlomar) [reply front-end/src/app/components/puzzle/puzzle.component.html]: done

File: front-end/src/app/components/device/device.component.html (hunk 2 above)
Comment (mrlomar) [reply front-end/src/app/components/device/device.component.html]: no spaces anymore

**Comment by GwennanS** [comment]
it saves uploaded configs still in the resources folder

**Pull request merged**